.github/workflows/lifecycle-image/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Stage 1
+FROM ubuntu:bionic AS download-and-extract-lifecycle
+
+ARG LIFECYCLE_URI
+
+WORKDIR /tmp
+
+RUN apt-get update && apt-get install -y wget
+RUN wget $LIFECYCLE_URI -O lifecycle.tgz && tar xzvf lifecycle.tgz
+
+RUN mkdir -p /cnb/lifecycle && mkdir /layers
```

Comment:
I believe this should be part of the second stage.

Reply:
We can't run these commands in distroless/static

Reply:
I tried chmod'ing `/layers` and then copying it over but it's still only writeable as root...

Reply:
lets just leave `/layers` how it is for now, it likely won't be a problem